comp/api/api/apiimpl/server_cmd.go

```diff
@@ -48,18 +48,21 @@ func (server *apiServer) startCMDServer(
 
 	// gRPC server
 	authInterceptor := grpcutil.AuthInterceptor(parseToken)
+	const maxMessageSize = 4 * 1024 * 1024 // 4 MB
 	opts := []grpc.ServerOption{
 		grpc.Creds(credentials.NewClientTLSFromCert(tlsCertPool, cmdAddr)),
 		grpc.StreamInterceptor(grpc_auth.StreamServerInterceptor(authInterceptor)),
 		grpc.UnaryInterceptor(grpc_auth.UnaryServerInterceptor(authInterceptor)),
+		grpc.MaxRecvMsgSize(maxMessageSize),
+		grpc.MaxSendMsgSize(maxMessageSize),
```

Comment:
the default message size in grpc is 4MB. 

I added these explicitly here in order to make sure the tagger server gets the same message max size when splitting the response into chunks.